src/main/java/gift/dto/ProductRequestDto.java

```diff
@@ -0,0 +1,36 @@
+package gift.dto;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+
+public class ProductRequestDto {
+
+    @NotBlank(message = "상품명은 필수입니다.")
+    private final String name;
+
+    @NotNull(message = "가격은 필수입니다.")
+    @Min(value = 0, message = "가격은 0원 이상입니다.")
+    private final Long price;
+
+    private String imageUrl;
```

Comment:
```suggestion
    private final String imageUrl;
```
통일성 있게 imageUrl 도 final 로 선언하시는 것은 어떨까요?